src/extension.ts

```diff
@@ -86,7 +86,14 @@ async function activate(context: vscode.ExtensionContext) {
     initializeState();
 
     status.updateNeedReplUi(false, context);
-    lsp.activate(context);
+
+
+    const controller = vscode.tests.createTestController('calvaTestController', 'Clojure Cider')
```

Comment:
Does `Clojure Cider` show up in the UI somewhere? Or what is it for?

Reply:
Good question. The first two arts are a globally unique ID and a human readable label. I haven’t seen the label in the UI, but I’ve just noticed that the name that I doesn’t have the word “Calva” in it.

Reply:
Here is the Go extension:

https://github.com/golang/vscode-go/blob/944fbf7185aa47cd3832bea776420996ae599e89/src/goTest/explore.ts#L37

Reply:
Yes, so that's why I am asking. It's probably best to give the label `Calva`. Wherever it shows up, that will make it clear  where it comes from. =) Not a blocker for merge though, imo.